flows/finance-data/finance_data_to_s3.py

```diff
@@ -0,0 +1,197 @@
+)
+def get_env_vars():
+    # Environment Variables from KV Store in Prefect
+    return get_key_value(key=f"atd_finance_data")
+
+
+@task(
+    name="pull_docker_image",
+    max_retries=1,
+    timeout=timedelta(minutes=60),
+    retry_delay=timedelta(minutes=5),
+    # state_handlers=[handler],
+)
+def pull_docker_image():
+    client = docker.from_env()
+    client.images.pull("atddocker/atd-finance-data", all_tags=True)
+    logger.info(docker_image)
+    return True
+
+
+@task(
+    name="upload_to_s3",
+    task_run_name="upload_to_s3: {name}",
+    # max_retries=1,
+    # timeout=timedelta(minutes=60),
+    # retry_delay=timedelta(minutes=5),
+    # state_handlers=[handler],
+)
+def upload_to_s3(environment_variables, name):
+    response = (
+        docker.from_env()
+        .containers.run(
+            image=docker_image,
+            working_dir=None,
+            command=f"python upload_to_s3.py {name}",
+            environment=environment_variables,
+            volumes=None,
+            remove=True,
+            detach=False,
+            stdout=True,
+        )
+        .decode("utf-8")
+    )
+    logger.info(response)
+    return response
+
+
+@task(
+    name="upload_to_knack",
+    task_run_name="upload_to_knack: {name}, {app}",
+    # max_retries=1,
+    # timeout=timedelta(minutes=60),
+    # retry_delay=timedelta(minutes=5),
+    # state_handlers=[handler],
+)
+def upload_to_knack(environment_variables, name, app, task_orders_res):
+    response = (
+        docker.from_env()
+        .containers.run(
+            image=docker_image,
+            working_dir=None,
+            command=f"python s3_to_knack.py {name} {app}",
+            environment=environment_variables,
+            volumes=None,
+            remove=True,
+            detach=False,
+            stdout=True,
+        )
+        .decode("utf-8")
+    )
+    logger.info(response)
+    return response
+
+
+@task(
+    name="upload_to_socrata",
+    task_run_name="upload_to_socrata",
+    # max_retries=1,
+    # timeout=timedelta(minutes=60),
+    # retry_delay=timedelta(minutes=5),
+    # state_handlers=[handler],
+)
+def upload_to_socrata(environment_variables, socrata_fl):
+    response = (
+        docker.from_env()
+        .containers.run(
+            image=docker_image,
+            working_dir=None,
+            command="python s3_to_socrata.py",
+            environment=environment_variables,
+            volumes=None,
+            remove=True,
+            detach=False,
+            stdout=True,
+        )
+        .decode("utf-8")
+    )
+    logger.info(response)
+    return response
+
+
+with Flow(
+    # Postfix the name of the flow with the environment it belongs to
+    f"Finance Data Publishing",
+    # Let's configure the agents to download the file from this repo
+    storage=GitHub(
+        repo="cityofaustin/atd-prefect",
+        path="flows/finance-data/finance_data_to_s3.py",
+        ref="ch-finance-data",  # The branch name
```

Comment:
do you want to switch this to main before you merge?

Reply:
will do thanks for the reminder 👍